include/cling/Interpreter/Interpreter.h

```diff
@@ -142,6 +142,11 @@ namespace cling {
     ///
     InvocationOptions m_Opts;
 
+    ///\brief Back pointer to *this* which stores parent information.
```

Comment:
This is necessary as putting a symbol into the JIT requires a constant location where it is available and **&this** is illegal.